tests/ops/functions/test_equal.py

```diff
@@ -834,9 +834,80 @@ def test_equal_simple_op_remaining(self):
             check_interface=False,
         )
 
-    def test_equal_with_different_arithmetic_depth(self):
-        """Test equal method with two operators with different arithmetic depth."""
-        assert not qml.equal(qml.adjoint(qml.PauliX(0)), qml.adjoint(qml.adjoint(qml.PauliX(0))))
```

Comment:
Why was this test deleted?

Reply:
Same as Albert.

Reply:
Oh by mistake, thanks for catching this!